### ISSUE TITLE
kill browser instance when using plugin

### ISSUE THREAD

**Issue by fantapop** [adieuadieu/serverless-chrome#149]
Is there a way to access this kill method when using the serverless-chrome plugin?
https://github.com/adieuadieu/serverless-chrome/blob/93f03eccee3be8e283f0a9b0a375186664f2eb40/packages/lambda/src/index.js#L107

When developing locally I can't get the browser to quit.

**Comment by adieuadieu** [comment]
Yes; an [object representing the chrome instance](https://github.com/adieuadieu/serverless-chrome/blob/93f03eccee3be8e283f0a9b0a375186664f2eb40/packages/lambda/src/index.js#L96-L116) is passed as a 4th argument to your lambda handler function.

When using the serverless-chrome-plugin with nodejs v8.10:

```javascript
module.exports.someLambdaHandler = async (event, context, callback, chromeInstance) => {
  // Chrome is already running!
  
  // do chrome-y stuff

  await chromeInstance.kill()

  return 'something'
}
```

**Comment by fantapop** [comment]
Okay, thank you.  It would be great to have the chromeInstance exposed on a global object so I don't need to modify my lambda signature.  We have shared code which builds our lambda handlers in a way that's abstracted from the specific api that's using it.  In order to support passing an additional parameter I needed to add it to the shared code that is used across all of our apis.

Also, why not expose a way to also relaunch the instance?  I was trying to include shutting down the instance in some error handling but I'm unable to start it again until the next lambda invocation.
